docker/ci/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     python3.10-dev \
     python3-pip
 
-RUN python3 -m pip install pyarrow
+RUN python3 -m pip install pyarrow requests
```

Comment:
Can we maintain a file such as `requirements.txt` to keep track of the Python modules? Then we can install the libs by `pip install -r requirements.txt`.
I think we should install numpy,pandas and scipy by default too.
The `requirements.txt` may look like:

```
numpy>=1.24
pandas>=1.5.3
scipy>=1.10.1
```

Reply:
sounds reasonable to me.